core/src/test/java/suites/VerySlowTestsSuite.java

```diff
@@ -21,16 +20,16 @@
 @Categories.IncludeCategory(VerySlowTests.class)
 @Categories.ExcludeCategory(StressTests.class)
 @Suite.SuiteClasses({
-    ParallelWorkerTests.class,
-    StandardApiTests.class,
-    MessageRepositoryTester.class,
-    PopReceiptTester.class,
-    QueueDeleterTests.class,
-    ReaderTester.class,
-    RepairTests.class,
-    TimeTests.class,
-    YamlConfigTest.class,
-    QueueRepositoryTester.class
-})
+                            ParallelWorkerTests.class,
```

Comment:
i think its lining up with the bracket. if we put the bracket on the next line it may clean up. i'll play with it. agree, its a lto of extra whitespace